libraries/AP_Compass/AP_Compass.cpp

```diff
@@ -1462,62 +1462,6 @@ void Compass::_detect_backends(void)
             }
 #endif
         }
-
-#if COMPASS_MAX_UNREG_DEV > 0
```

Comment:
I think a COMPASS_OPTIONS bit, maybe "DisableDroneCANReplacement" which can be set on vehicles where the GPS/compass combos are powered separately from the main flight control MCU